src/mobu/jupyterclient.py

```diff
@@ -31,6 +37,16 @@
 __all__ = ["JupyterClient"]
 
 
+@dataclass(frozen=True)
+class JupyterLabSession:
+    """This holds the information a client needs to talk to the Lab in order
+    to execute code."""
+
+    session_id: str = ""
+    kernel_id: str = ""
+    websocket: Optional[ClientWebSocketResponse] = None
```

Comment:
I don't think any of these should have default values. You always pass in all three, so they can be mandatory.

Reply:
Done